src/ontology/mondo-edit.obo

```diff
@@ -466845,14 +466837,18 @@ is_a: MONDO:0024286 {source="NCIT:C27503"} ! benign blood vessel neoplasm
 
 [Term]
 id: MONDO:0024462
```

Comment:
Are the changes in this class to address the comment in the issue about which parent term to use?

Reply:
yes!